grpc/src/main/java/com/avast/metrics/grpc/GrpcServerMonitoringInterceptor.java

```diff
@@ -27,15 +28,17 @@ public <ReqT, RespT> ServerCall.Listener<ReqT> interceptCall(final ServerCall<Re
 
         cache.getMeter(method, "Calls", "count").mark();
 
-        final AtomicInteger currentCalls = cache.getGaugedValue(method, "Current");
```

Comment:
Ta stavajici `Current` zmizi, vadi nam to?

Reply:
I´m OK with this change.